pkgs/development/python-modules/drawille/default.nix

```
@@ -0,0 +1,19 @@
+{lib, buildPythonPackage, fetchPypi}:
```

```suggestion
{ lib
, buildPythonPackage
, fetchPypi
}:
```

One per line to simplify diffs. Same for the other packages.